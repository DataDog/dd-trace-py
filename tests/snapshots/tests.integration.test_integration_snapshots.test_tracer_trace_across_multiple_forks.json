[[
  {
    "name": "parent",
    "service": "",
    "resource": "parent",
    "trace_id": 0,
    "span_id": 1,
    "parent_id": 0,
    "type": "",
    "error": 0,
    "meta": {
      "_dd.p.dm": "-0",
      "language": "python",
      "runtime-id": "d5f3e666fe79454c90ec7063f44787b5"
    },
    "metrics": {
      "_dd.agent_psr": 1.0,
      "_dd.top_level": 1,
      "_dd.tracer_kr": 1.0,
      "_sampling_priority_v1": 1,
      "process_id": 19794
    },
    "duration": 17556566,
    "start": 1691163615434076624
  },
     {
       "name": "child1",
       "service": "",
       "resource": "child1",
       "trace_id": 0,
       "span_id": 2,
       "parent_id": 1,
       "type": "",
       "error": 0,
       "meta": {
         "language": "python",
         "runtime-id": "60f937fe2fa5494c8aeceab4b18a5174"
       },
       "metrics": {
         "_dd.top_level": 1,
         "_dd.tracer_kr": 1.0,
         "_sampling_priority_v1": 1,
         "process_id": 19882
       },
       "duration": 8826524,
       "start": 1691163615437388850
     },
        {
          "name": "child2",
          "service": "",
          "resource": "child2",
          "trace_id": 0,
          "span_id": 3,
          "parent_id": 2,
          "type": "",
          "error": 0,
          "meta": {
            "language": "python",
            "runtime-id": "0a98b167e74c408abbb58db3fd26a2e7"
          },
          "metrics": {
            "_dd.top_level": 1,
            "_dd.tracer_kr": 1.0,
            "_sampling_priority_v1": 1,
            "process_id": 19883
          },
          "duration": 57412,
          "start": 1691163615440398294
        }]]
