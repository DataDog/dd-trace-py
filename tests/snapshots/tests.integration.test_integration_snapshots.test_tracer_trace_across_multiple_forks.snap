[[{"name" "parent"
   "service" nil
   "resource" "parent"
   "error" 0
   "span_id" 0
   "trace_id" 0
   "parent_id" nil
   "start" 1624045173599735000
   "duration" 51650000
   "meta" {"runtime-id" "120a8105b0c542d889b939d7c91d71fa"}
   "metrics" {"_dd.agent_psr" 1.0
              "_sampling_priority_v1" 1
              "system.pid" 12266
              "_dd.tracer_kr" 1.0}}
      {"name" "child1"
       "service" nil
       "resource" "child1"
       "error" 0
       "span_id" 1
       "trace_id" 0
       "parent_id" 0
       "start" 1624045173606288000
       "duration" 32138000
       "meta" {"runtime-id" "8fba36822c6846d895a13f5d043543be"}
       "metrics" {"_sampling_priority_v1" 1
                  "system.pid" 12294
                  "_dd.tracer_kr" 1.0}}
          {"name" "child2"
           "service" nil
           "resource" "child2"
           "error" 0
           "span_id" 2
           "trace_id" 0
           "parent_id" 1
           "start" 1624045173612023000
           "duration" 86000
           "meta" {"runtime-id" "859c494affcb4933a1e15b6c4d2561b8"}
           "metrics" {"_sampling_priority_v1" 1
                      "system.pid" 12295
                      "_dd.tracer_kr" 1.0}}]]
